### PR TITLE
Declare query and holepunch before reference to ontimeout

### PR DESCRIPTION
**Pull request by 4c656554**
Re issue #44

### CODE REVIEW

**Comment by mafintosh** [comment]
In ontimeout you need to check if holepunch and query are set as well before destroying them, ie

```
if (holepunch) holepunch.destroy()
```

**Comment by 4c656554** [comment]
Made a new commit...

**Comment by mafintosh** [comment]
Awesome, will merge on green ci :)

File: index.js [hunk 1]
```diff
@@ -116,6 +116,8 @@ module.exports = class HyperDHT extends DHT {
 
   async connectRaw (publicKey, opts = {}) {
     let error = null
+    let query
+    let holepunch
```
Comment (mafintosh) [inline index.js:120]: standard wants you to initialise these to null

**Pull request merged**